administration/src/bp-modules/cards/hooks/useCardGenerator.ts

```diff
@@ -170,15 +170,18 @@ const useCardGenerator = (region: Region): UseCardGeneratorReturn => {
     [cards, client, projectConfig, handleError, sendCardConfirmationMails, region.activatedForCardConfirmationMail]
   )
 
+  const filenameSaveString = (input: string) =>
+    input.replace(/ /g, "_").normalize('NFKD').replace(/[^a-z,A-Z,_]/g, '')
```

Comment:
:wrench: Depending on which PR is merged first, it would be cool if we could reuse the functionality from #1782 instead of re-implementing it here.

Reply:
good point, as the other PR is already merged, i will update mine